modules/authentication/proc-enabling-authentication-with-github.adoc

```diff
@@ -175,6 +139,56 @@ auth:
         sessionDuration: { hours: 24 }
 ----
 
+`signIn` ::
+
+`resolvers`:::
+After successful authentication, the user signing in must be resolved to an existing user in the {product-short} catalog. To best match users securely for your use case, consider configuring a specific resolver. Enter the resolver list to override the default resolver: `usernameMatchingUserEntityName`.
```

Comment:
```suggestion
After successful authentication, the signed-in user needs to be resolved to an existing user in the {product-short} catalog. To ensure a secure and accurate user match for your setup, you must configure a specific resolver. You can define the resolver list to override the default `usernameMatchingUserEntityName` resolver.
```

Reply:
The suggestion is less accurate:

* First sentence: present tense is accurate, past tense is not.
* Second sentence: "consider" is accurate, "must" is not.
* Third sentence: "Enter" follows style guide: https://redhat-documentation.github.io/supplementary-style-guide/#text-entry